desktop-app/app/components/ErrorBoundary/index.js

```diff
@@ -26,20 +41,19 @@ export default class ErrorBoundary extends React.Component {
       return (
         <Fragment>
           <h1 style={{textAlign: 'center', marginTop: 100}}>
-            Something went wrong.
+            App has crashed!
           </h1>
-          <div style={{display: 'flex', justifyContent: 'center'}}>
-            <pre
-              style={{overflow: 'scroll', background: '#6d6d6d', width: '80%'}}
-            >
-              {JSON.stringify(this.state, null, 2)}
-            </pre>
-          </div>
-          <p style={{width: '80%', textAlign: 'center'}}>
+          <p style={{textAlign: 'center'}}>
+            Stack Trace: <TextBoxWithCopyButton text={this.state.error} />
```

Comment:
I personally don't like this as a one line textboxes. 
Despite being an error and that all users do not have to be devs, I would like to be able to read and review this information in a simple way. I would use textareas as big as possible with a font like Consolas and selectable text, and of course add the copy button (and maybe a download to file button) in each one

Something like this
![app crash](https://user-images.githubusercontent.com/13673443/91626521-1095f380-e9b0-11ea-8009-7eb30485eb15.png)

Maybe this is something we need to discuss @gorogoroumaru  with @manojVivek @jzabala @esprush on Slack

Reply:
Yeah, we can change the one line textbox to a text area with copy button. But we can skip the download to not increase the scope of this. We can add it in the future if that becomes a necessity.

For now
1. TextArea with copy button
2. `Create Issue` button creating an issue on GitHub with the issue body populated with app `About` details, `stack trace` and `error info` should be sufficient.

Reply:
yeah, let's not increase the scope of this, sorry 😅
We could use [collapsible sections](https://gist.githubusercontent.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab/raw/9b0b7be74b312af333b9aeea289f7dc42bd3b85a/markdown-details-collapsible.md) for the JSONs

Reply:
Hi I'm reporting an app crash:
>Version: 0.11.0
>Electron: 9.1.1
>Chrome: 83.0.4103.122
>Node.js: 12.14.1
>V8: 8.3.110.13-electron.0
>OS: Windows_NT x64 10.0.18362

<details>
  <summary>Error Info</summary>
  
  ```json
  {
    "key1": "value1",
    "key2": "value2"
  }
  ```
</details>

<details>
  <summary>Stack Trace</summary>
  
  ```json
  {
    "key1": "value1",
    "key2": "value2"
  }
  ```
</details>